### DISCUSSION TITLE
[FEATURE REQUEST] Consulta sobre los nombres finales de los PB Bot v3

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#140] [3. Ideas & Feature Requests] [answered]
Hola tengo la siguiente duda:

Los nombre de los PB bot estarán en inglés para cualquier idioma de la laptop? (si mi laptop está en español, me arrojaba los nombres en ingles), ó si está en español estará en español los nombres?.

Lo pregunto porque antes estaban los nombres en español (ejemplo solicitudes de red) y ahora está en ingles (network request).

Igual no me hace sentido, si existen los dos idiomas en la configuración (debería decir new stage que nueva etapa si es que es inglés para todo)
Debo un ss.

![image](https://user-images.githubusercontent.com/61419471/116928707-6865b800-ac2b-11eb-9da6-1e11625f840a.png)

Gracias por la ayuda :)

**Comment by NicolasPLG** [comment, marked as answer]
Me auto respondo:

Corresponde al idioma que tiene configurado la compañía. Si la compañía está en ingles, todo lo demás estará en Ingles. aunque la duda que tengo entonces, qué debe pasar con los nombres que aparecen en español?

**Reply by acedward** [reply]
@NicolasPLG si la documentación de los bots ahora depende de la idoma de la compañía. 
Esto será corregido en el futuro y dependerá del idioma del dispositivo del usuario.

**Reply by NicolasPLG** [reply]
ok @acedward  muchas gracias pr tu respuesta
